internal/backends/database.go

```diff
@@ -83,7 +84,14 @@ type ListCollectionsResult struct {
 
 // CollectionInfo represents information about a single collection.
 type CollectionInfo struct {
-	Name string
+	Name            string
+	CappedSize      int64 // TODO https://github.com/FerretDB/FerretDB/issues/3458
+	CappedDocuments int64 // TODO https://github.com/FerretDB/FerretDB/issues/3458
+}
+
+// Capped returns true if collection is capped.
+func (ci *CollectionInfo) Capped() bool {
+	return ci.CappedSize > 0 || ci.CappedDocuments > 0
```

Comment:
According to this https://mongodb.com/docs/manual/core/capped-collections/#create-a-capped-collection size is always set.

> The size field is always required, even when you specify the max number of documents. MongoDB removes older documents if a collection reaches the maximum size limit before it reaches the maximum document count.

Reply:
It is always set for capped collections. If both `CappedSize` and `CappedDocuments` are zero, `Capped()` returns false because that collection is not capped

Reply:
Thanks :+1: